lib/express-rate-limit.js

```diff
@@ -48,20 +48,31 @@ function RateLimit(options) {
         var key = options.keyGenerator(req, res);
 
         options.store.incr(key, function(err, current) {
-          if (err) {
-            return next(err);
-          }
+            if (err) {
+              return next(err);
+            }
 
-          if (options.max && current > options.max) {
-              return options.handler(req,res, next);
-          }
+            if (options.headers) {
+              res.setHeader('X-RateLimit-Limit', options.max);
+              res.setHeader('X-RateLimit-Remaining', ((options.max - current) > 0) ? (options.max - current) : 0);
+            }
 
-          if (options.delayAfter && options.delayMs && current > options.delayAfter) {
+            req.rateLimit = ({
```

Comment:
Hey, sorry I missed that you had updated this PR. You don't need parenthesis here, but I'm going to go ahead and merge this anyways